docs/guides/slices-pattern.md

```diff
@@ -62,6 +62,37 @@ function App() {
 export default App
 ```
 
+### Changing multiple stores together
+
+You can affect multiple stores together in a single function itself.
```

Comment:
What do you think about:

"You can update multiple stores, at the same time, in a single function."

 - "affect" may hold a slightly different meaning (for example, like that in useEffect or a side effect)
 - "itself" feels redundant at the end of the sentence. 

In terms of the heading, do you think something succinct like:

"Updating multiple stores"

Thoughts @Rec0iL99 ?

Reply:
After thinking about it, yes I agree.